Source/WebKit/Shared/EditingRange.cpp

```diff
@@ -35,7 +35,6 @@ namespace WebKit {
 
 std::optional<WebCore::SimpleRange> EditingRange::toRange(WebCore::LocalFrame& frame, const EditingRange& editingRange, EditingRangeIsRelativeTo base)
 {
-    ASSERT(editingRange.location != notFound);
```

Comment:
Can't we just return std::nullopt when it's notFound? What's the point of running the rest of the function?

Reply:
When it is notFound, the behavior is just to return the whole range of the editable content, which is different than returning null opt (which results in just nothing happening and the caller returning an empty string, which is not desired)

Reply:
(ie, the function still does something and returns a meaningful range)

Reply:
In that case, we should explain that in the commit message.